hs/src/Reach/AST/DLBase.hs

```diff
@@ -1058,7 +1088,7 @@ instance Pretty a => Pretty (DLRecv a) where
       <> render_nest (pretty dr_k)
 
 data FluidVar
-  = FV_balance Int
+  = FV_balances Integer (M.Map DLArg Int)
```

Comment:
What's the integer and the map? I thought this was going to just be `FV_balances` (no parameter)